### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by tasneemtoolba**
- Code style update (formatting, renaming)

Resolves: issue #916

## Summary by CodeRabbit

- **Documentation**
	- Enhanced the "How to get started" section of the README with troubleshooting steps for installation issues, including commands for clearing the npm cache and retrying installation.
	- Added alternative installation commands (`npm i --legacy-peer-deps` and `npm i --force`) with cautionary notes on potential dependency conflicts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ npm i
 npm run dev
 ```
 
+If you faced problems with ```npm i ```, you may try ```npm i --force``` instead
```
[the remark this answers is not in the document]
Comment (fricoben) [reply README.md]: Please use AI recommendations and add the possibility to use `npm i --legacy-peer-deps`

**Review by fricoben — CHANGES_REQUESTED** [review]
Need better readme here. AI recommended a better proposition

**Comment by tasneemtoolba** [comment]
@fricoben how to trigger CodeRabbitai on an already existing PR ?) or you're another AI app xD?)) ~~jk~~

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**